lib/browser/methods/devices.rb

```diff
@@ -37,5 +37,9 @@ def playbook?
     def windows_touchscreen_desktop?
       windows? && !!(ua =~ /Touch/)
     end
+    
+    def desktop?
+      !mobile? && !tablet?
```

Comment:
Ah good spot I didn't see that, I guess this could be the same for tv. Maybe needs a rethink